torch/csrc/jit/register_prim_ops.cpp

```diff
@@ -436,6 +436,15 @@ RegisterOperators reg({
           };
         }),
 
+    Operator(
+        prim::Error,
```

Comment:
This seems overly general. The error message should clearly tell the user what went wrong (they tried to execute an `@ignored` function) and how to fix it, possibly with a reference to the node's source location. Also, prim::Error seems too generic a name